packages/tokamak/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -170,6 +170,14 @@ abstract contract CrossDomainMessenger is
         OTHER_MESSENGER = _otherMessenger;
     }
 
+    /// @notice Sends a deposit ton message to some target address on the other chain. Note that if the call
+    ///         always reverts, then the message will be unrelayable, and any ETH sent will be
+    ///         permanently locked. The same will occur if the target on the other chain is
+    ///         considered unsafe (see the _isUnsafeTarget() function).
+    function sendTONMessage(address , uint256 , bytes calldata , uint32 ) external virtual {
```

Comment:
I think we can delete this function. we don't have to override

Reply:
Thank you! 
I updated!